### DISCUSSION TITLE
咨询问题

### DISCUSSION THREAD

**Discussion by mc15829702687** [zouyaoji/vue-cesium#306] [Q&A]
大佬，我想咨询个问题，我画了个箭头，然后，我想箭头刚开始渲染的时候，有个生长动画效果，这应该怎么实现，有么有现成的动画组件

![微信图片_20221101162936](https://user-images.githubusercontent.com/44308726/199191352-e36e84ac-9da6-42a9-bd92-a62f1ec0906e.jpg)

**Comment by zouyaoji** [comment]
抱歉了 还没抽出精力去集成这一块。

**Reply by mc15829702687** [reply]
没事，大佬，就是遇到这个需求卡在这了，想咨询下你
